gargoyle-postgresql/src/Gargoyle/PostgreSQL.hs

```diff
@@ -53,7 +53,7 @@ initLocalPostgres binPath dbDir = do
     , "-U", "postgres"
     , "--no-locale"
     , "-E", "UTF8"
-    ]) { std_in = CreatePipe, std_out = CreatePipe, std_err = CreatePipe }
+    ]) { std_in = NoStream, std_out = NoStream, std_err = NoStream }
```

Comment:
I tried getting a minimal repro to exhibit the SIGPIPE crashes from using CreatePipe, but was not successful

Reply:
Also thoughts of `Inherit`ing std_err to actually see the errors and not just the exit code or reading the handle and printing it's output on ExitFailure

Reply:
That sounds reasonable to me; not sure whether there are any potential downsides.